### DISCUSSION TITLE
some bugs identified from the latest update

### DISCUSSION THREAD

**Discussion by Rasb74** [rern/rAudio#1337] [Bug Reports]
Power button still not working.
Pdf file viewer not working 
JSON parse when go to a view back on iPhone, working perfectly on the other devices

**Comment by ceejayemm** [comment]
I can add to this list:

- after adding 2 new radio stations, which were updated to the Shared Data area, it is not possible to add any artwork as Change Cover Art did NOT produce any dialogue box for this to be done - but the stations were usable, just no station icons. Error below is generated  and can be seen in journalctl -xe:

Jun 05 20:07:22 rAudio-1 nginx[379]: 2023/06/05 20:07:22 [error] 379#379: *107 open() "/srv/http/data/webradio/img/http:||192.168.86.250:9790|minimstreamer|*|H_R2|-thumb.jpg" failed (2: No such file or directory), client: 192.168.86.53, server: localhost, request: "GET /data/webradio/img/http:%7C%7C192.168.86.250:9790%7Cminimstreamer%7C*%7CH_R2%7C-thumb.jpg?v=1685992043

it seems default images are not created when the station is added.
EDIT

It seems that when Shared Data is enabled the owner/group on the Web Radio folder/files changes from http:http to mpd:mpd as shown here:

With Shared Data DISABLED:
rAudio-1:~  #  ls -lrt /srv/http/data/webradio/
total 24
-rw-r--r-- 1 http http   54 Aug 18  2020 'https:||stream.radioparadise.com|rock-flac'
-rw-r--r-- 1 http http   54 Aug 18  2020 'https:||stream.radioparadise.com|flac'
-rw-r--r-- 1 http http   48 Jul 19  2021 'https:||stream.radioparadise.com|mellow-flac'
-rw-r--r-- 1 http http   56 Nov 27  2022 'https:||stream.radioparadise.com|global-flac'
drwxr-xr-x 2 http http 4096 May 21 09:38  img
-rw-r--r-- 1 http http   27 May 30 10:08 'http:||a.files.bbci.co.uk|media|live|manifesto|audio|simulcast|hls|uk|sbr_high|ak|bbc_radio_two.m3u8'

With Shared Data ENABLED:
rAudio-1:~  #  ls -lrt /srv/http/data/webradio
lrwxrwxrwx 1 http http 26 Jun  6 09:42 /srv/http/data/webradio -> /mnt/MPD/NAS/data/webradio
rAudio-1:~  #  ls -lrt /srv/http/data/webradio/
total 20
drwxr-xr-x 2 mpd mpd  0 Jun  6 09:42  img
-rwxr-xr-x 1 mpd mpd 54 Jun  6 09:42 'https:||stream.radioparadise.com|rock-flac'
-rwxr-xr-x 1 mpd mpd 48 Jun  6 09:42 'https:||stream.radioparadise.com|mellow-flac'
-rwxr-xr-x 1 mpd mpd 56 Jun  6 09:42 'https:||stream.radioparadise.com|global-flac'
-rwxr-xr-x 1 mpd mpd 27 Jun  6 09:42 'http:||a.files.bbci.co.uk|media|live|manifesto|audio|simulcast|hls|uk|sbr_high|ak|bbc_radio_two.m3u8'
-rwxr-xr-x 1 mpd mpd 54 Jun  6 09:42 'https:||stream.radioparadise.com|flac'


- changing Playback screen options is hit and miss, I can turn bars on and off but I can't turn off the Time display for example (or Buttons or Artwork).
- Library options - cannot enable 'Album - sort by Artist'
- Features / Others / Autoplay - with Bluetooth Connected enabled, connecting my Bluetooth speaker does not start audio playback as it used to to in earlier versions.

Chris

**Reply by ceejayemm** [reply]
All of the items I added above still seem to be a problem with 20230609.

Chris
